3-platform/warehouse/db_postgres/src/main/resources/db/changelog/changes/03-create-inventories-table.sql

```diff
@@ -0,0 +1,24 @@
+-- inventories table
+
+CREATE TABLE public.inventories
+(
+    id                 varchar NOT NULL,
+    warehouse_id       varchar NOT NULL,
+    product_id         varchar NULL,
+    actual_quantity    int     NULL,
+    available_quantity int     NULL,
+    CONSTRAINT inventories_pk PRIMARY KEY (id),
+    CONSTRAINT inventories_actual_quantity_check CHECK (actual_quantity >= 0),
+    CONSTRAINT inventories_available_quantity_check CHECK (available_quantity >= 0),
+    CONSTRAINT inventories_check CHECK (available_quantity <= actual_quantity),
+    CONSTRAINT inventories_warehouses_fk FOREIGN KEY (id) REFERENCES public.warehouses (id),
+    CONSTRAINT inventories_products_fk FOREIGN KEY (id) REFERENCES public.products (id)
```

Comment:
what do you think about adding these constraints (except PK, FK) separately via `alter table add constraints`?
```suggestion
    CONSTRAINT inventories_actual_quantity_check CHECK (actual_quantity >= 0),
    CONSTRAINT inventories_available_quantity_check CHECK (available_quantity >= 0),
    CONSTRAINT inventories_check CHECK (available_quantity <= actual_quantity),
    CONSTRAINT inventories_warehouses_fk FOREIGN KEY (id) REFERENCES public.warehouses (id),
    CONSTRAINT inventories_products_fk FOREIGN KEY (id) REFERENCES public.products (id)
```

Reply:
Ok, let's separate that checks and paste them after table creation.